scripts/deploy/main.bicep

```diff
@@ -14,7 +14,12 @@ param webAppServiceSku string = 'B1'
 
 @description('Location of package to deploy as the web service')
 #disable-next-line no-hardcoded-env-urls
-param packageUri string = 'https://aka.ms/copilotchat/webapi/latest'
+param webApiPackageUri string = 'https://aka.ms/copilotchat/webapi/latest'
+
+@description('Location of package to deploy as the memory pipeline')
```

Comment:
Again, there is no memory pipeline, there is a memory service!

Reply:
we could go for memory pipeline service? But will stick with pipeline for now until things stabilize.